MVVMRxSwiftDemo/Sources/Presentation/Navigator/Navigator+Scene.swift

```diff
@@ -14,6 +14,8 @@ extension Navigator {
         case login(viewModel: LoginViewModelProtocol)
         case home(viewModel: HomeViewModelProtocol)
         case surveyDetail(viewModel: SurveyDetailViewModelProtocol)
+        case surveyQuestion(viewModel: SurveyQuestionViewModelProtocol)
+        case yesCancelAlert(title: String, message: String, yesHandler: AlertCompletion?)
```

Comment:
As I mentioned about, so I would prefer something like this

```suggestion
        case alert(viewModel: AlertViewModel)
```